Lib/test/test_platform.py

```diff
@@ -552,6 +552,7 @@ def test_libc_ver(self):
                 (b'GLIBC_2.9', ('glibc', '2.9')),
                 (b'libc.so.1.2.5', ('libc', '1.2.5')),
                 (b'libc_pthread.so.1.2.5', ('libc', '1.2.5_pthread')),
+                (b'/aports/main/musl/src/musl-1.2.5', ('musl', '1.2.5')),
```

Comment:
Can you add tests on version with 2 numbers and 4 numbers? Like `/aports/main/musl/src/musl-1.2` and `/aports/main/musl/src/musl-1.2.5.1`.

Reply:
I could, but I don't believe musl will ever have such release numbers, so is it worth doing?

Reply:
Parsing a binary file using a regular expression is a strange thing for me. So I prefer to have tests to make sure that the implementation is reliable. The question is more if the regex will match versions with 2 members (x.y) or 4 members (x.y.z.a), or if it only matchs version with 3 members (x.y.z).

Reply:
Fair enough, though the existing tests don't do that, nor do they test the no-match cases.  I could add such tests, but at that point I'd want to rewrite this test method to split it up into multiple tests, and I don't think that's in scope for this PR :)  I'll add the couple of extra checks, though.